code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -620,7 +620,9 @@
 
 	flags_gun_features = GUN_INTERNAL_MAG|GUN_WIELDED_FIRING_ONLY
 	gun_skill_category = GUN_SKILL_SPEC
+	reload_sound = 'sound/weapons/gun_mortar_reload.ogg'
 	var/datum/effect_system/smoke_spread/smoke
+	var/obj/item/ammo_magazine/rocket/loaded_rocket = null
```

Comment:
What's the difference between this and `current_mag` aka `/obj/item/ammo_magazine/internal/launcher/rocket`?

Reply:
The internal magazine is essentially a magazine of magazines.

Reply:
Why making another magazine var then?